docs/src/user-docs/guides-quick-start.md

```diff
@@ -4,6 +4,7 @@ The guide below briefly describes how to get started with running a task on Spid
 you'll need to:
 
 * Write a task
+* (Optional) Install Dependencies
```

Comment:
```suggestion
* Install Spider's dependencies
```

Installing Spider's dependencies isn't actually optional. Using the task is the optional part.